### PR TITLE
Fix 2FA Login

### PR DESCRIPTION
**Pull request by shantur**

### CODE REVIEW

**Comment by TheGui01** [comment]
Hello,
I've just test it and it works !! Thanks so much.

I noticed 1 change and 1 issue :
in the configuration.yaml I had to put my code under quote : 
code: '1234' instead of code: 1234 

The issue I have is that HA detects 2 new integration : 
1 named with my Verisure username which works
1 named Securitas direct which is detected new at each reboot and that we cannot ignore. It has a configure button but the 2FA check fails

**Comment by shantur** [comment]
@TheGui01 - Thanks for testing.
I limited my changes to just fix the Login. I believe that other changes/ fixes would be part of an ongoing work that @guerrerotook might be doing

File: custom_components/securitas/securitas_direct_new_api/apimanager.py [hunk 1]
```diff
@@ -82,7 +82,7 @@ async def _execute_request(self, content, operation: str) -> ClientResponse:
                 "id": self._generate_id(),
                 "country": self.country,
                 "lang": self.language,
-                "callby": "OWP_10",
```
Comment (guerrerotook) [inline custom_components/securitas/securitas_direct_new_api/apimanager.py:85]: I didn't notice this small change in the string. Good catch!

**Review by guerrerotook — APPROVED** [review]

**Comment by guerrerotook** [comment]
Thanks very much!

I'm going to merge that right now and test in on my local environment to see how it works!

**Pull request merged**